python/src/skale_contracts/projects/predeployed.py

```diff
@@ -0,0 +1,44 @@
+"""Module connects predeployed project to the SKALE contracts library"""
+from __future__ import annotations
+from typing import TYPE_CHECKING
+from eth_utils.address import to_canonical_address
+
+from skale_contracts.instance import Instance
+from skale_contracts.project import Project
+
+if TYPE_CHECKING:
+    from eth_typing import Address, ChecksumAddress
+
+
+class PaymasterInstance(Instance):
+    """Represents instance of paymaster"""
+
+    def get_contract_address(
+            self,
+            name: str,
+            *args: str | Address | ChecksumAddress
+    ) -> Address:
+        if name in {"Paymaster", "FastForwardPaymaster"}:
+            return self.address
+        return to_canonical_address(
```

Comment:
It's better to use `elif` and call `authority()` only for `PaymasterAccessManager`.
If unknown name is passed an exception should be raised.